src/main/java/FindCommand.java

```diff
@@ -0,0 +1,30 @@
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+
+/**
+ * class that handles the command "find"
+ * that searches the storage list and stores
+ * them in the find list.
+ */
+public class FindCommand {
+
+    public ArrayList<Task> findList;
+
+    public FindCommand() {
+        this.findList = new ArrayList<>();
+    }
+
+    public void find(TaskList tasks, String keyword) {
+        for (int i = 0; i < tasks.storage.size(); i++) {
+            if (tasks.storage.get(i).getDescription().contains(keyword)) {
```

Comment:
Would creating a String variable to store tasks.storage.get(i).getDescription() before the 'if' statement be better? 

You may also consider creating:
Task task = tasks.storage.get(i)
String description = task.getDescription() 
:)